api/v1alpha1/envoyproxy_types.go

```diff
@@ -134,7 +134,7 @@ type EnvoyProxyKubernetesProvider struct {
 	// +optional
 	// +kubebuilder:validation:XValidation:message="minReplicas must be greater than 0",rule="!has(self.minReplicas) || self.minReplicas > 0"
 	// +kubebuilder:validation:XValidation:message="maxReplicas must be greater than 0",rule="!has(self.maxReplicas) || self.maxReplicas > 0"
-	// +kubebuilder:validation:XValidation:message="maxReplicas cannot be less than or equal to minReplicas",rule="!has(self.minReplicas) || self.maxReplicas > self.minReplicas"
+	// +kubebuilder:validation:XValidation:message="maxReplicas cannot be less than minReplicas",rule="!has(self.minReplicas) || self.maxReplicas >= self.minReplicas"
```

Comment:
do we have any user case here? 

i don't think setting `maxReplicas` equal to `minReplicas` is necessary. if we do, what's point of using HPA? suggest using replicas setting directly.
https://github.com/envoyproxy/gateway/blob/c2f88f0e9be766d3cd8eb75c686277ebf2da1b43/api/v1alpha1/shared_types.go#L58

any inputs? cc @envoyproxy/gateway-maintainers

Reply:
No specific use case, I just think maxReplicas should be aligned with the spec.

Reply:
I suggest leaving the current validation as it is unless we have a strong reason to change it.